network/manager.go

```diff
@@ -143,6 +149,18 @@ func (nm *networkManager) restore(isRehydrationRequired bool) error {
 		}
 	}
 
+	// Parsing the Stdin the CRI will pass to CNI
+	// this is to leverage it in Delete/Create network during the initial CNI rehydration
+	stdinData, err := io.ReadAll(os.Stdin)
```

Comment:
@tamilmani1989 reading in stdin to recreate the CNI config here to ensure when we restore/rehydrate we can leverage the timeout , let me know if see any issues with that

Reply:
lets remove this code..its not required..rehydration will be always false for cni..also you cannot read stdin twice..it can be read only once

Reply:
removed the code